src/Apps/Conversations/components/ConversationCTA/ConversationMakeOfferButton.tsx

```diff
@@ -77,6 +78,7 @@ export const ConversationMakeOfferButton: React.FC<ConversationMakeOfferButtonPr
         )
       }
     } catch (error) {
+      sendToast({ message: "Error", variant: "error" })
```

Comment:
The way we have it elsewhere in volt is like: `Error. Please try again.`

(Ditto below)

Reply:
Not for this PR but i feel like with our structured mutation success/error types it should be so much easier to implement consistent error handling that we can add to with the general fallback message. Maybe there's already a good pattern somewhere.

Reply:
Really? I feel like it's the opposite. Because now you have to query for the error. Previously, errors are just baked into everything.

Reply:
done in https://github.com/artsy/force/pull/13952/commits/095bf0ccffb6896fd8102faf7bd962c9cf55c5e5